model_analyzer/config/generate/config_parameters.py

```diff
@@ -53,12 +55,147 @@ def get_type(self, name: str) -> ParameterType:
     def get_category(self, name: str) -> ParameterCategory:
         return self._parameters[name].category
 
-    def get_range(self, name: str) -> Tuple[int, int]:
+    def get_range(self, name: str) -> Tuple[Optional[int], Optional[int]]:
         return (self._parameters[name].min_range, self._parameters[name].max_range)
 
-    def get_list(self, name: str) -> List[Any]:
+    def get_list(self, name: str) -> Optional[List[Any]]:
         return self._parameters[name].enumerated_list
 
+    def populate_search_parameters(
+        self,
+        config: ConfigCommandProfile,
+        parameters: Dict[str, Any],
+        model_config_parameters: Dict[str, Any],
+    ) -> None:
+        self._populate_parameters(config, parameters)
+        self._populate_model_config_parameters(config, model_config_parameters)
+
+    def _populate_parameters(
+        self,
+        config: ConfigCommandProfile,
+        parameters: Dict[str, Any],
+    ) -> None:
+        self._populate_parameter(
+            parameter_name="batch_sizes",
+            rcs_parameter_min_value=config.run_config_search_min_model_batch_size,
```

Comment:
We talked about potentially breaking this into two populate parameters (RCS vs list parameter)

Reply:
Broken!